napari_allencell_annotator/view/annotator_view.py

```diff
@@ -230,11 +238,17 @@ def render_values(self, vals: list[Any]):
             the values for the annotations.
         """
         for item, annotation in zip(self.annot_list.items, vals):
-            item_data = annotation
-            if item_data is None or item_data == "":
+            if annotation is None or annotation == "":
                 item.set_default_value()
             else:
-                item.set_value(item_data)
+                if item.type == ItemType.POINT:
+                    annot_name: str = item.name.text()
+                    self._annotator_model.add_points_layer(
+                        annot_name, self.viewer.create_points_layer(annot_name, True, annotation)
+                    )
+                else:
```

Comment:
`elif` `else` here instead of nested

Reply:
with comments

Reply:
elif and common in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/87b951e137cb0b43e2ecc0cebeeea66a28e0c981